cache.go

```diff
@@ -87,15 +90,37 @@ func (cd *cachedData) flush() {
 		return
 	}
 
-	file, err := os.OpenFile(cd.Path, os.O_RDWR|os.O_TRUNC, 0666)
+	file, err := fileutil.LockFile(cd.Path, os.O_RDWR, 0666)
 	if err != nil {
 		log.Printf("Cannot write to the cache file: %v", err)
```

Comment:
`LockFile` is blocking therefore a different error message would be unnecessary since it wouldn't fail because of concurrency issues.